pipeline.yaml

```diff
@@ -1,14 +1,19 @@
 # `pipeline.yaml` is the main configuration file for an MLflow Pipeline.
 # Required pipeline parameters should be defined in this file with either concrete values or
 # variables such as {{ INGEST_DATA_LOCATION }}.
-
+#
 # Variables must be dereferenced in a profile YAML file, located under `profiles/`.
 # See `profiles/local.yaml` for example usage. One may switch among profiles quickly by
 # providing a profile name such as `local` in the Pipeline object constructor:
 # `p = Pipeline(profile="local")`
 #
+# NOTE: All "FIXME::REQUIRED" fields in pipeline.yaml and profiles/*.yaml must be set correctly
+#       to adapt this template to a specific regression problem. To find all required fields,
+#       under the root directory of this pipeline, type on a unix-like command line:
+#       $> grep "# FIXME::REQUIRED:" pipeline.yaml profiles/*.yaml
+#
 # NOTE: YAML does not support tabs for indentation. Please use spaces and ensure that all YAML
-# files are properly formatted.
+#       files are properly formatted.
```

Comment:
Nit: Is the indentation change intended?

Reply:
Yes. I want to make "NOTE" to stand out.